src/types/permissions.d.ts

```diff
@@ -30,3 +30,12 @@ export interface IdpGroup {
   name: string;
   groups: string[]; // these should be names of lxd groups
 }
+
+export interface LxdTrustToken {
+  client_name: string;
+  addresses: string[];
+  expires_at: string;
+  fingerprint: string;
+  type: string;
```

Comment:
we can be more specific with the `type` field since it has limited number of possible values